src/liquid/components/ld-notification/ld-notification.css

```
@@ -0,0 +1,159 @@
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  min-width: 20rem;
+  font: var(--ld-typo-body-m);
+  pointer-events: none;
+}
+
+@keyframes ld-notification-fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes ld-notification-fade-out {
+  from {
+    transform: none;
+    opacity: 1;
+  }
+  to {
+    transform: none;
+    opacity: 0;
+  }
+}
+
+.ld-notification__item {
+  pointer-events: auto;
+  position: absolute;
+  transform-origin: center;
+  box-shadow: var(--ld-shadow-sticky);
+  width: fit-content;
+  min-width: 19rem;
+  max-width: calc(90% - 1rem);
+  max-height: 80vh;
+  min-height: 2.5rem;
+  overflow-y: auto;
+  border-radius: var(--ld-br-m);
+  display: flex;
+  justify-content: space-between;
+  transition: transform 0.2s ease, opacity 0.2s linear;
+
+  &:where(:not(.ld-notification__item--dismissed)) {
+    animation: ld-notification-fade-in 0.2s linear;
+  }
+
+  &:nth-last-of-type(n + 4) {
+    display: none;
+  }
+
+  &:nth-last-of-type(n + 2) {
+    .ld-notification__item-content,
+    .ld-notification__btn-dismiss {
+      opacity: 0;
+    }
+  }
+
+  &:nth-last-of-type(2) {
+    opacity: 0.75;
+  }
+
+  &:nth-last-of-type(3) {
+    opacity: 0.5;
+  }
+
+  &:nth-last-of-type(4) {
+    opacity: 0.25;
+  }
+
+  &.ld-notification__item--dismissed {
+    opacity: 0;
+    animation: ld-notification-fade-out 0.2s linear forwards;
+  }
+
+  .ld-notification--bottom & {
+    bottom: var(--ld-sp-32);
+
+    &:nth-last-of-type(2) {
+      transform: scale(0.975) translateY(25%);
+    }
+
+    &:nth-last-of-type(3) {
+      transform: scale(0.95) translateY(50%);
+    }
+
+    &:nth-last-of-type(4) {
+      transform: scale(0.925) translateY(75%);
+    }
+  }
+
+  .ld-notification--top & {
+    top: var(--ld-sp-32);
+
+    &:nth-last-of-type(2) {
+      transform: scale(0.975) translateY(-25%);
+    }
+
+    &:nth-last-of-type(3) {
+      transform: scale(0.95) translateY(-50%);
+    }
+
+    &:nth-last-of-type(4) {
+      transform: scale(0.925) translateY(-75%);
+    }
+  }
+}
+
+.ld-notification__item--info {
+  background-color: var(--ld-col-rb-default);
+  color: var(--ld-col-wht);
+}
+.ld-notification__item--warn {
+  background-color: var(--ld-col-vy-default);
+  color: var(--ld-col-rblck-default);
+}
+.ld-notification__item--alert {
+  background-color: var(--ld-col-rr-default);
+  color: var(--ld-col-wht);
+}
+
+.ld-notification__item-content,
+.ld-notification__btn-dismiss {
+  transition: opacity 0.2s linear;
+}
+
+.ld-notification__item-content {
+  display: grid;
+  grid-auto-flow: column;
+  align-items: center;
+  gap: var(--ld-sp-12);
+  padding: var(--ld-sp-6) var(--ld-sp-12) var(--ld-sp-8);
+}
+
+.ld-notification__btn-dismiss {
+  margin-left: var(--ld-sp-16);
+  align-self: center;
+  flex-shrink: 0;
+  border: 0;
+  border-radius: var(--ld-br-full);
+  background-color: transparent;
+  cursor: pointer;
+  padding: 0;
+  height: 2.5rem;
+  width: 2.5rem;
```

```suggestion
  height: var(--ld-sp-40);
  width: var(--ld-sp-40);
```

I know it's size and not spacing. As an alternative, maybe we could add custom css vars for size...?